geomstats/_backend/tensorflow/__init__.py

```diff
@@ -817,8 +846,13 @@ def tile(x, multiples):
     return tf.tile(x_reshape, multiples)
 
 
+def vec_to_diag(vec):
```

Comment:
Put on top of the file as, see:
https://github.com/geomstats/geomstats/blob/1b2c2cc47c5bccdfb0236adf8246b942f934d4e8/geomstats/_backend/tensorflow/__init__.py#L74